lib/rspec/matchers.rb

```diff
@@ -745,6 +745,38 @@ def raise_error(error=Exception, message=nil, &block)
       desc.sub("raise", "raising")
     end
 
+    # Matches if an expectation fails
+    #
+    # @example
+    #
+    #   expect { some_expectation }.to fail
+    def fail
+      raise_error(RSpec::Expectations::ExpectationNotMetError)
+    end
+
+    # Matches if an expectation fails with the provided message
+    #
+    # @example
+    #
```

Comment:
Please remove these blank lines between `@example` and the example code.  YARD assumes the code starts on the immediate next line and when you have a blank line it renders it, too, which is unnecessary and looks odd.